src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -29,10 +29,13 @@
 	public static int LEFT_LEAD_CHANNEL;
 	public static int LEFT_FOLLOWER_1_CHANNEL;
 	public static int LEFT_FOLLOWER_2_CHANNEL;
-
+	public static boolean LEFT_SENSOR_IS_INVERTED;
```

Comment:
Put the LEFT_SENSOR_IS_INVERTED variable and the RIGHT_SENSOR_IS_INVERTED variable near each other. Also, put DRIVE in the name somewhere to make it very clear, something like INVERT_LEFT_DRIVE_SENSOR.